src/xenia/base/debugging_posix.cc

```diff
@@ -20,16 +20,9 @@ namespace debugging {
 bool IsDebuggerAttached() { return false; }
 void Break() { raise(SIGTRAP); }
 
-void DebugPrint(const char* fmt, ...) {
-  StringBuffer buff;
-
-  va_list va;
-  va_start(va, fmt);
-  buff.AppendVarargs(fmt, va);
-  va_end(va);
-
-  // OutputDebugStringA(buff.GetString());
-}
+namespace internal {
+void DebugPrint(const char* s) {}
```

Comment:
Maybe at least add a TODO comment.

Reply:
Didn't change the implementation, but good idea for the TODO comment.